pkg/ring/lifecycler_test.go

```diff
@@ -35,6 +43,8 @@ func testLifecyclerConfig(ringConfig Config, id string) LifecyclerConfig {
 	lifecyclerConfig.NumTokens = 1
 	lifecyclerConfig.ID = id
 	lifecyclerConfig.FinalSleep = 0
+	lifecyclerConfig.TokensFilePath = tokenDir
```

Comment:
Do we need it in the default config used for tests? I would say we want it into in the specific test we have introduced for the tokens persistency.

Reply:
I just realized that this doesn't make sense at all. I am deleting the directory in the `defer` :man_facepalming:. I will modify to only include it in the new test.